src/GovUk.Education.ExploreEducationStatistics.Data.Api/Models/PermalinkEntity.cs

```diff
@@ -0,0 +1,21 @@
+namespace GovUk.Education.ExploreEducationStatistics.Data.Api.Models
```

Comment:
Is there any reason why you added this to Data.Api.Models and not Data.Model?

Reply:
Good spot, i forgot about that

Reply:
Though thinking about it this isnt exactly the Data.Model, as all the logic is contained within the Data.Api. Happy either way though